tsconfig.json

```diff
@@ -2,7 +2,10 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "dist",
+    "moduleResolution": "node",
+    "module": "ES2015",
+    "allowSyntheticDefaultImports": true
```

Comment:
I would prefer to keep `allowSyntheticDefaultImports` disabled unless we *really* need it.  It's far too easy to break downstream modules with that flag set.

Reply:
Okay.
hmh, without it calling `assert` and `nock` did not work anymore.
I have to check again, if I can make it work with that flag 🤔

Reply:
done